ogusa/wealth.py

```diff
@@ -1,37 +1,92 @@
+import os
 import numpy as np
 import pandas as pd
-import os
+from ogusa import utils
 
 CUR_PATH = os.path.split(os.path.abspath(__file__))[0]
 
 
-def get_wealth_data():
+def get_wealth_data(scf_yrs_list=[2016, 2013, 2010, 2007], web=True,
+                    directory=None):
     '''
-    Reads wealth data from the 2007, 2010, and 2013 Survey of Consumer
-    Finances (SCF) files.
+    Reads wealth data from the 2007, 2010, 2013, and 2016 Survey of
+    Consumer Finances (SCF) files.
 
     Args:
-        None
+        scf_yrs_list (list of 4-digit integers): list of SCF years to
```

Comment:
I think just make this type `(list)` for documentation purposes (so consistent with elsewhere). The part after the colon can describe the need for four digit integers if that's not obvious from the other description.